FirmataParser.cpp

```diff
@@ -194,16 +207,35 @@ int FirmataParser::setDataBufferOfSize(uint8_t * dataBuffer, size_t dataBufferSi
  * DIGITAL_MESSAGE, REPORT_ANALOG, REPORT DIGITAL, SET_PIN_MODE and SET_DIGITAL_PIN_VALUE).
```

Comment:
Can you describe a description of when and why to use the context parameter? Also is this a required parameter, and if not, what is the default value?